### DISCUSSION TITLE
route().current(name) not working as expected in 2.0.4

### DISCUSSION THREAD

**Discussion by BamButz** [tighten/ziggy#728] [Q&A] [answered]
The README says that route().current('events.*') should return true when current URI is /events.
After upgrading from 1.8.2 to 2.0.4 this is not true anymore it is returning false now.
If I change the wildcard to route().current('events*') it works. 

Is this expected? I couldn't find anything regarding this in the Upgrade section.

**Comment by bakerkretzmar** [comment, marked as answer]
The argument to `current()` is the route **name**, so if your `/events` route is named `events.index` it should work. If it's just named `events` I'm not sure... you're saying that worked fine in v1 and doesn't in v2?

**Reply by BamButz** [reply]
Oh okay, that would not be the problem for me to change the route name and yes it actually worked in v1.
